test/general/brainpool.js

```diff
@@ -184,7 +184,7 @@ EJ4QcD/oQ6x1M/8X/iKQCtxZP8RnlrbH7ExkNON5s5g=
     const pk = await openpgp.key.readArmored(data[name].priv);
     expect(pk).to.exist;
     expect(pk.getKeyId().toHex()).to.equal(data[name].id);
-    expect(await pk.decrypt(data[name].pass)).to.be.true;
+    await expect(pk.decrypt(data[name].pass)).to.not.be.rejected;
```

Comment:
I would just write
```suggestion
    await pk.decrypt(data[name].pass);
```
(and similarly elsewhere); if it throws, the test will fail also.